### PR TITLE
Create A New Reminder

### LINKED ISSUE #7: Create A New Note
At the bottom of the `reminders.hbs` template, there should be a button that routes the user to `/reminders/new` where users can create a new reminder.

* The new reminder should have inputs for the title, date, and notes.
* The new reminder should have a button for submitting the new note.
* Pressing the button should submit a new reminder to the database and immediately displays in the reminders list.
**Hot Tip**: There are no tests set up for this user story. So, you’ll have to write an acceptance test.

### PR DESCRIPTION
**Pull request by adam-rice**
@martensonbj @Tman22 @dylanavery720 

## Purpose

This request adds the ability for a user to view a form on the reminders/new route, where they can add a new reminder by inputing the title, date, and notes. The user is able to submit the form via the 'Add' button which immediately adds the reminder to their list.

fixes #7 💥 🔫 

## Approach

We added the necessary route and created the component to view the form. There is an action attached to the form which triggers the components js file to update the store and also reformat the date coming from the input field.

### Learning

We had to research how to create an action and link the action to the appropriate component.js file. Also, we investigated formatting the date as originally it was just coming back as a string. Lastly, we learned how to test the functionality with an acceptance test using q-unit.

### Open Questions and Pre-Merge TODOs

- [ ] Use Github checklists. When solved, check the box and explain the answer.

### Test coverage 

The test associated with this pull request shows that a user can click the Create New Reminder and the route is updated to /new, then the user can click 'Add' and their reminder class count for all reminders is increased by one.

### Follow-up tasks

- [Issue #8](https://github.com/turingschool-projects/1610-remember-2/issues/8)

### CODE REVIEW

File: app/components/reminder-item.js [hunk 1]
```diff
@@ -1,6 +0,0 @@
-import Ember from 'ember';
```
Comment (martensonbj) [inline app/components/reminder-item.js:1]: 👍

**Review by martensonbj — CHANGES_REQUESTED** [review]
Remove that routes test please. Super clean though! Well done guys

**Comment by martensonbj** [comment]
Love the consistancy with PR title and commit messages - probably don't need to have the word 'wip' in the the commit message

**Comment by adam-rice** [comment]
@martensonbj @dylanavery720 

Commit 7be10de pushed that removes the unit/route test for 'new'

**Review by martensonbj — APPROVED** [review]

**Comment by martensonbj** [comment]
:shipit:

**Pull request merged**